supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesCallPaths.scala

```diff
@@ -13,7 +13,7 @@ object JesCallPaths {
 }
 
 class JesCallPaths(jobKey: BackendJobDescriptorKey, workflowDescriptor: BackendWorkflowDescriptor, jesConfiguration: JesConfiguration,
-                   val gcsFileSystem: GcsFileSystem) extends JesWorkflowPaths(workflowDescriptor, jesConfiguration, gcsFileSystem) {
+                   gcsFileSystem: GcsFileSystem) extends JesWorkflowPaths(workflowDescriptor, jesConfiguration, gcsFileSystem) {
```

Comment:
Because I made it a `val` in `JesWorkflowPaths`, which `JesCallPaths` extends from, so there was no need to have it here (I actually think I remember getting an error for not overriding it or something).